reference/docs-conceptual/dev-cross-plat/vscode/using-vscode-for-debugging-compiled-cmdlets.md

```diff
@@ -0,0 +1,204 @@
+        "preLaunchTask": "build",
+        "program": "pwsh",
+        "args": [
+            "-NoExit",
+            "-NoProfile",
+            "-Command",
+            "Import-Module ${workspaceFolder}/myModule/bin/Debug/netstandard2.0/myModule.dll",
+        ],
+        "cwd": "${workspaceFolder}",
+        "stopAtEntry": false,
+        "console": "integratedTerminal"
+    }
+   ```
+
+The `program` field is used to launch `pwsh` so that the cmdlet being debugged can be run. The
+`-NoExit` argument prevents the PowerShell session from exiting as soon as the module is imported.
+The path in the `Import-Module` argument is the default build output path when you've followed the
+[Writing Portable Modules][] guide. If you've created a module manifest (`.psd1` file), you should
+use the path to that instead. The `/` path separator works on Windows, Linux, and macOS. You must
+use the integrated terminal to run the PowerShell commands you want to debug.
+
+### Configuring launch.json for Windows PowerShell
+
+This launch configuration works for testing your cmdlets in Windows PowerShell (`powershell.exe`).
+Create a second launch configuration with the following changes:
+
+1. `name` should be `PowerShell cmdlets: powershell`
+
+1. `type` should be `clr`
+
+1. `program` should be `powershell`
+
+   It should look like this:
+
+   ```json
+    {
+        "name": "PowerShell cmdlets: powershell",
+        "type": "clr",
+        "request": "launch",
+        "preLaunchTask": "build",
+        "program": "powershell",
+        "args": [
+            "-NoExit",
+            "-NoProfile",
+            "-Command",
+            "Import-Module ${workspaceFolder}/myModule/bin/Debug/netstandard2.0/myModule.dll",
+        ],
+        "cwd": "${workspaceFolder}",
+        "stopAtEntry": false,
+        "console": "integratedTerminal"
+    }
+   ```
+
+## Launching a debugging session
+
+Now everything is ready to begin debugging.
+
+- Place a breakpoint in the source code for the cmdlet you want to debug:
+
+  ![A breakpoint shows as a red dot in the gutter](media/using-vscode-for-debugging-compiled-cmdlets/set-breakpoint.png)
+
+- Ensure that the relevant **PowerShell cmdlets** configuration is selected in the configuration
+  drop-down menu in the **Debug** view:
+
+  ![Select the launch configuration](media/using-vscode-for-debugging-compiled-cmdlets/select-launch-configuration.png)
+
+- Press <kbd>F5</kbd> or click on the **Start Debugging** button
+
+- Switch to the terminal pane and invoke your cmdlet:
+
+  ![Invoke the cmdlet](media/using-vscode-for-debugging-compiled-cmdlets/invoke-the-cmdlet.png)
+
+- Execution stops at the breakpoint:
+
+  ![Executions halts at breakpoint](media/using-vscode-for-debugging-compiled-cmdlets/stopped-at-breakpoint.png)
+
+You can step through the source code, inspect variables, and inspect the call stack.
+
+To end debugging, click **Stop** in the debug toolbar or press <kbd>Shift</kbd>-<kbd>F5</kbd>. The
```

Comment:
Same question on this shortcut... does it apply to all platforms?

Reply:
Same on all.